### PR TITLE
Added email enumeration to github module and util class for issue #203

### PR DESCRIPTION
**Pull request by vikas-kundu**
Fixes Issue #203 

#### Resolves

Adding email enumeration based on user using github api

#### Changes

- core/util/github
- modules/search/github

#### Checklist

- [x] I have read the Contribution & Best practices Guideline.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The acceptance, integration, unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by vikas-kundu** [comment]
Here is a GIF of the same: https://asciinema.org/a/guLlyIPehXCdvAKov3LtgHgA5

**Comment by z3y50n** [comment]
It seems to be working fine

**Comment by saeeddhqan** [comment]
Could you add this feature to the email_search as well?

**Comment by vikas-kundu** [comment]
> Could you add this feature to the email_search as well?

Working on it

**Comment by vikas-kundu** [comment]
> Could you add this feature to the email_search as well?

Done in this commit https://github.com/saeeddhqan/Maryam/pull/204/commits/4077a797b9c97db04c38a1b6166627b9086b07e4

File: modules/osint/email_search.py [hunk 1]
```diff
@@ -69,10 +76,20 @@ def module_api(self):
...
 		'ask': f"%40{domain}",
-		'hunter': f"{domain}&api_key={key}"
+		'hunter': f"{domain}&api_key={key}",
+		'github': domain
 	}
 	self.thread(search, self.options['thread'], engines, query, q_formats, limit, count, meta['sources'])
 	output = {'emails': list(set(EMAILS))}
+	
+	temp_set = set()
+	for user, email_data in GIT_RAW_EMAIL_DATA.items():
```
Comment (saeeddhqan) [inline modules/osint/email_search.py]: this loop has been duplicated. so write it in /util/github at once.

**Comment by saeeddhqan** [comment]
does it work fine after my change?

**Comment by vikas-kundu** [comment]
> does it work fine after my change?

Yes. See image: https://ibb.co/kSQTq4F

**Pull request merged**